### PR TITLE
UBER-504: fix presenters on ListItem. Add DeviceSizes.

### PR DESCRIPTION
**Pull request by SasLord**
# Contribution checklist

## Brief description

Снимок экрана 2023-06-27 в 02 05 43

Снимок экрана 2023-06-27 в 23 51 58

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: plugins/view/src/index.ts [hunk 1]
```diff
@@ -535,8 +535,8 @@ export interface DisplayProps {
   key?: string
   excludeByKey?: string
   fixed?: 'left' | 'right' // using for align items in row
+  suffix?: boolean
```
Comment (BykhovDenis) [inline plugins/view/src/index.ts:538]: why we need it?

File: packages/ui/src/components/internal/Root.svelte [hunk 2]
```diff
@@ -100,6 +100,44 @@
   document.addEventListener('dblclick', (event) => {
     event.preventDefault()
   })
+
+  let remove: any = null
+  const getSize = (width: number): 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl' => {
```
Comment (haiodo) [inline packages/ui/src/components/internal/Root.svelte]: Please make it simplier
```typescript
type WidthType = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl'
const getSize = (width: number): WidthType => {
  const widths = [480, 680, 760, 1024, 1208, 1209]
  const sizes: WidthType[] = ['xs', 'sm', 'md', 'lg', 'xl', 'xxl']
  return sizes[widths.findIndex(it => it > width)]
}
```

File: packages/ui/src/components/internal/Root.svelte [hunk 3]
```diff
@@ -100,6 +100,44 @@
...
+          : width <= 1024
+            ? 'lg'
+            : width <= 1208
+              ? 'xl'
+              : 'xxl'
+  }
+  const updateDeviceSize = () => {
+    if (remove !== null) remove()
+    const size = getSize(docWidth)
+    const mqString =
```
Comment (haiodo) [inline packages/ui/src/components/internal/Root.svelte]:
```typescript
const css: Record<WidthType, string> = {
  xs: '(max-width: 480px)',
  sm: '(min-width: 480.01px) and (max-width: 680px)',
  md: '(min-width: 680.01px) and (max-width: 760px)',
  lg: '(min-width: 760.01px) and (max-width: 1024px)',
  xl: '(min-width: 1024.01px) and (max-width: 1208px)',
  xxl: '(min-width: 1208.01px)'
}
const mqString = css[size]
````

File: packages/ui/src/types.ts [hunk 4]
```diff
@@ -301,6 +301,7 @@ export interface DeviceOptions {
   isPortrait: boolean
   isMobile: boolean
   fontSize: number
+  size: 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl' | null
```
Comment (haiodo) [inline packages/ui/src/types.ts]: Please use WidthType

File: plugins/tags-resources/src/components/TagsReferencePresenter.svelte [hunk 5]
```diff
@@ -0,0 +1,119 @@
+<!--
+// Copyright © 2022 Hardcore Engineering Inc.
```
Comment (haiodo) [inline plugins/tags-resources/src/components/TagsReferencePresenter.svelte]:
```suggestion
// Copyright © 2023 Hardcore Engineering Inc.
```

File: plugins/view/src/index.ts (hunk 1 above)
Comment (SasLord) [reply plugins/view/src/index.ts:538]: For correct display of AttchmentsPresenter and CommentsPresenter in compact mode

**Review by haiodo — APPROVED** [review]

**Pull request merged**